### ISSUE TITLE
Define function calls in forms

### ISSUE THREAD

**Issue by budde377** [budde377/Part#97]
The same way that `AjaxServer` has been successfull serverside, we can create similar clientside. By using our dear language functionString we can define the function call in the formular,  and call registered instances in the server. We can also define what notions to set on success or on error (maybe even with error-number). 

---

Imported from Bitbucket
Originally reported by: Christian Budde Christensen (2014-10-22T03:55:48.287)

**Comment by budde377** [comment]
Example: 

```
<form data-function-string='{object}.{method}({name-of-element}, ... )
```

---

Original comment by: Christian Budde Christensen (2014-10-22 01:57:06+00:00)

**Comment by budde377** [comment]
Created Function String parser for client side.
See #97

→ <<cset 16e0076b6584>>

---

Original comment by: Christian Budde Christensen (2014-11-19 03:31:13+00:00)

**Comment by budde377** [comment]
Fixed #97

→ <<cset 8c02652d13e8>>

---

Original comment by: Christian Budde Christensen (2014-11-21 19:32:48+00:00)
